### PR TITLE
11 grafana

### PR DESCRIPTION
**Pull request by aweberlopes**
docker-compose is updated with a influxdb config and automated creation of datasource in grafana and automated dashboard creation

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -13,9 +13,6 @@ services:
       DBHOST: influx
       DBUSER: bwm
       DBPASSWORD: bwm
-    volumes:
```
Comment (racoon63) [inline docker-compose.yml]: Why are you not creating local volumes instead of docker volumes for better portability?

File: docker-compose.yml [hunk 2]
```diff
@@ -2,82 +2,50 @@ version: '3.1'
...
+      - INFLUXDB_PASSWORD=bwm
   grafana:
     image: grafana/grafana:latest
     ports:
       - '3000:3000'
+    networks:
+      - bwm
     volumes:
       - grafana-storage:/var/lib/grafana
       - ./grafana-provisioning/:/etc/grafana/provisioning
```
Comment (racoon63) [inline docker-compose.yml:33]: Can this be included in one of the docker volumes? If yes, please do so.

**Review by racoon63 — COMMENTED** [review]
@aweberlopes please have a look on this. Dammit github reviews :D

File: docker-compose.yml (hunk 2 above)
Comment (aweberlopes) [reply docker-compose.yml:33]: grafana-provisioning held config file for data source and dashboards. On start time grafana takes it and load it. 
Advantage you can change it localy and then restart the docker-compose and the dashboard will be changed automaticaly

**Review by racoon63 — APPROVED** [review]
Nice! Thank you for contributing m8!

**Pull request merged**